openedx/openedx-platform
# refactor: Move CourseEditLTIFieldsEnabledFlag to xblock-lti-consumer [BD-38] [TNL-8104]

This moves the CourseEditLTIFieldsEnabledFlag to xblock-lti-consumer. An intended side-effect of this is that this model is now available in the LMS in additon to Studio. 
The other part of this refactor is in this PR: https://github.com/edx/xblock-lti-consumer/pull/159
This PR should include a version bump of xblock-lti-consumer that includes the change from that PR, but that requires that PR to merge first. 

It partially addresses EDUCATOR-121. 

**JIRA tickets**:  EDUCATOR-121

**Dependencies**: https://github.com/edx/xblock-lti-consumer/pull/159 

**Sandbox URL**: TBD - sandbox is being provisioned.

**Merge deadline**: "None" 

**Testing instructions**:

1. Switch to this branch
2. Update xblock-lti-consumer to use the PR linked above
3. Run migrations
4. There should be no errors.
5. There should be no data loss in the CourseEditLTIFieldsEnabledFlag model
6. CourseEditLTIFieldsEnabledFlag should now be accessible via LMS as well.

**Reviewers**

**Settings**
```yaml
EDXAPP_FEATURES:
  ENABLE_COMBINED_LOGIN_REGISTRATION: true
```
